public/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
+    <meta name="description" content="Web based Honey Framework UI - for live trading and executing algorithmic orders and strategies">
```

Comment:
;ets remove strategies word until we add Strategy Editor page hosted version

```suggestion
    <meta name="description" content="Web based Honey Framework UI - for live trading and executing algorithmic orders">
```